pkg/pipeline/templates/terraformed.go.tmpl

```diff
@@ -23,13 +22,21 @@ func (tr *{{ .CRD.Kind }}) GetTerraformResourceIDField() string {
 	return "{{ .Terraform.IdentifierField }}"
 }
 
+// GetConnectionDetailsMapping for this {{ .CRD.Kind }}
+func (tr *{{ .CRD.Kind }}) GetConnectionDetailsMapping() map[string]string {
+  return map[string]string{ {{range $k, $v := .SensitiveFields}}"{{ $k }}": "{{ $v}}", {{end}} }
+}
+
 // GetObservation of this {{ .CRD.Kind }}
 func (tr *{{ .CRD.Kind }}) GetObservation() (map[string]interface{}, error) {
 	o, err := json.TFParser.Marshal(tr.Status.AtProvider)
 	if err != nil {
 		return nil, err
 	}
 	base := map[string]interface{}{}
+	if err := json.TFParser.Unmarshal(o, &base); err != nil {
+	  return nil, err
+	}
 	return base, json.TFParser.Unmarshal(o, &base)
```

Comment:
```suggestion
	return base, json.TFParser.Unmarshal(o, &base)
```

Seems like this is remnant of the earlier implementation.

Reply:
Ah good catch! I can swear that I deleted them and not sure how they came back, possibly some ctrl+z mistake